### DISCUSSION TITLE
New Quire exhibition from Leventhal Center: "Processing Place"

### DISCUSSION THREAD

**Discussion by itspangler** [thegetty/quire#953] [Show & Tell]
I'm pleased to share the latest exhibition from the Leventhal Map & Education Center, _Processing Place: How Computers and Cartographers Redrew Our World_.

The show will be on display in our gallery until March 2025, and the Quire publication will be hosted permanently on our website: https://www.leventhalmap.org/digital-exhibitions/processing-place/

Here is the source repo: https://github.com/bplmaps/processing-place

**Comment by Erin-Cecele** [comment]
This is fantastic! Thank you for sharing @itspangler. We'd love to add this to our showcase! Can you please fill out this form: https://forms.gle/qpV7fjmVACfxJ4zE6 so we can share your work with our community?
